open-apparel-registry/open-apparel-registry
# Mark failing reports with Do Not Open

## Overview

Two reports (Percent Facilities With Multiple Matches, and
Average Affiliations) timeout and cause site outages.

We are updating their names to reflect that they should not be opened
until such time as we have time for a full fix of the reports.

Connects #1389 

## Demo

<img width="434" alt="Screen Shot 2022-05-05 at 9 15 08 AM" src="https://user-images.githubusercontent.com/21046714/166931716-1125e900-2a15-44b6-99df-feee411a940e.png">

## Testing Instructions

* Run `./scripts/server` and login as c1@example.com
* View [reports](http://localhost:8081/admin/reports/) and confirm that the two reports have 'Do Not Open' at the beginning of their names.

## Checklist
